<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(40, 55, 57);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="30,70">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,40,10,50">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="20,20,20,20">
          <item>
           <widget class="QPushButton" name="prev_window">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 63 8pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);
padding: 8px;
bottom-margin:3px;
</string>
            </property>
            <property name="text">
             <string>PREV</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveButton">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 63 8pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);
padding: 8px;
bottom-margin:3px;
</string>
            </property>
            <property name="text">
             <string>SAVE</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveAsButton">
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 63 8pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);
padding: 8px;
bottom-margin:3px;
</string>
            </property>
            <property name="text">
             <string>SAVE AS</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="loadButton">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 63 8pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);
padding: 8px;
bottom-margin:3px;
</string>
            </property>
            <property name="text">
             <string>LOAD</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="5,90,5">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_15">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5" stretch="5,85">
            <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">font: 63 9pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);
border: 0px solid blue;
</string>
              </property>
              <property name="text">
               <string>Poles and Zeros Diagram</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="GraphWidget2" name="z_p_diagram" native="true"/>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_16">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="20,0">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="5,90,5">
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QLabel" name="title_combo_1">
                <property name="styleSheet">
                 <string notr="true">font: 63 6pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);

</string>
                </property>
                <property name="text">
                 <string>Title 1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="combo1">
                <property name="styleSheet">
                 <string notr="true">font: 63 9pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);

</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="title_combo_2">
                <property name="styleSheet">
                 <string notr="true">font: 63 6pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);

</string>
                </property>
                <property name="text">
                 <string>Title 2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="combo2">
                <property name="styleSheet">
                 <string notr="true">font: 63 9pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);

</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="5,90,5">
            <item>
             <spacer name="horizontalSpacer_8">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QPushButton" name="create_button">
                <property name="styleSheet">
                 <string notr="true">font: 63 9pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>Create</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_5">
                <property name="styleSheet">
                 <string notr="true">font: 63 6pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);
border: 0px solid blue;
</string>
                </property>
                <property name="text">
                 <string>or</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="autostage">
                <property name="styleSheet">
                 <string notr="true">font: 63 9pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>Auto-Stage (Max RD)</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="5,90,5">
          <item>
           <spacer name="horizontalSpacer_14">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3" stretch="85,15">
            <item>
             <widget class="QGroupBox" name="param_group_box">
              <property name="styleSheet">
               <string notr="true">color:rgb(255, 255, 255)</string>
              </property>
              <property name="title">
               <string>Parameters</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label_6">
                <property name="styleSheet">
                 <string notr="true">font: 63 8pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);
border: 0px solid blue;</string>
                </property>
                <property name="text">
                 <string>Gain</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="gainSpin">
                <property name="styleSheet">
                 <string notr="true">font: 63 7pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);</string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignHCenter">
               <widget class="QLabel" name="label">
                <property name="styleSheet">
                 <string notr="true">font: 63 8pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);
border: 0px solid blue;</string>
                </property>
                <property name="text">
                 <string>dB</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="goGain">
                <property name="styleSheet">
                 <string notr="true">font: 63 9pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>Set Gain</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_13">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="rightLayout" stretch="0,55,4,1,35,4">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="20,5,5,5,10,5,5,5,10,10,5,10,5,0">
          <item>
           <spacer name="horizontalSpacer_17">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_14">
            <property name="styleSheet">
             <string notr="true">font: 63 8pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);
border: 0px solid blue;</string>
            </property>
            <property name="text">
             <string>Vmin</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QDoubleSpinBox" name="vminSpin">
            <property name="styleSheet">
             <string notr="true">font: 63 8pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);
border: 0px solid blue;</string>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="value">
             <double>0.020000000000000</double>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft">
           <widget class="QLabel" name="label_13">
            <property name="styleSheet">
             <string notr="true">font: 63 8pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);
border: 0px solid blue;</string>
            </property>
            <property name="text">
             <string>V</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_18">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_12">
            <property name="styleSheet">
             <string notr="true">font: 63 8pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);
border: 0px solid blue;</string>
            </property>
            <property name="text">
             <string>Vmax</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="vmaxSpin">
            <property name="styleSheet">
             <string notr="true">font: 63 8pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);
border: 0px solid blue;</string>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="value">
             <double>15.000000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="styleSheet">
             <string notr="true">font: 63 8pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);
border: 0px solid blue;</string>
            </property>
            <property name="text">
             <string>V</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_19">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="reloadv">
            <property name="styleSheet">
             <string notr="true">font: 63 9pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Calculate DR</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_20">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_10">
            <property name="styleSheet">
             <string notr="true">font: 63 11pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);
border: 0px solid blue;</string>
            </property>
            <property name="text">
             <string>RD:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="rd">
            <property name="styleSheet">
             <string notr="true">font: 63 11pt &quot;Open Sans Semibold&quot;;
color:rgb(255, 255, 255);
border: 0px solid blue;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_21">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="10,80,10">
          <item>
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="GraphWidget" name="graphWidget" native="true"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_10">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QGroupBox" name="showing_group">
            <property name="styleSheet">
             <string notr="true">color:rgb(255, 255, 255)</string>
            </property>
            <property name="title">
             <string>Showing</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="stagesLayoutaa">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QScrollArea" name="stagesLayout">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>703</width>
               <height>245</height>
              </rect>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="35,10,10,10,10,10,35">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="backButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_11">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="deleteButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_12">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="nextButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphWidget</class>
   <extends>QWidget</extends>
   <header location="global">FrontEnd/UIs/graphwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GraphWidget2</class>
   <extends>QWidget</extends>
   <header location="global">FrontEnd/UIs/graphwidget2.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
